CONTRIBUTING.md

```diff
@@ -15,8 +15,8 @@ contribution. See the [DCO](DCO) file for details.
 # Email and Chat
 
 The project currently uses the general CoreOS email list and IRC channel:
```

Comment:
Also, this sentence with `the general CoreOS email list and IRC channel` needs to be modified according to the proposed changes. Thanks!

Reply:
Hi @spzala,

Thanks for your comment. It should be in synced with the info in main repo now.